wasm/minver/zboxsdk_min.go

```diff
@@ -0,0 +1,90 @@
+package minver
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	// "sync"
+	"syscall/js"
+
+	// "github.com/0chain/gosdk/core/zcncrypto"
+	"github.com/0chain/gosdk/zboxcore/sdk"
+)
+
+// convert JS String to []String
+func ZBOXstrToListSring(s string) []string {
+	slice := []string{}
+	err := json.Unmarshal([]byte(s), &slice)
+
+	if err != nil {
+		panic(err)
+	}
+	return slice
+}
+
+func strToPriceRange(s string) sdk.PriceRange {
+	var p sdk.PriceRange
+	err := json.Unmarshal([]byte(s), &p)
+	if err == nil {
+		fmt.Println("error:", err)
+	}
+
+	return p
+}
+
+func strToBlob(s string) sdk.Blobber {
+	var b sdk.Blobber
+	err := json.Unmarshal([]byte(s), &b)
+	if err == nil {
+		fmt.Println("error:", err)
+	}
+
+	return b
+}
+
+func InitAuthTicket(this js.Value, p []js.Value) interface{} {
+	authTicket := p[0].String()
+	result := sdk.InitAuthTicket(authTicket)
+	return result
+}
+
+func ZBOXGetClientEncryptedPublicKey(this js.Value, p []js.Value) interface{} {
+	result, err := sdk.GetClientEncryptedPublicKey()
```

Comment:
@Rizary Have you tested this calls? I am sure they will not work as they access http calls.

Reply:
Same for functions below

Reply:
I will update the min version which already updated in the wasm package.